client/app/calendar/tasks/hooks/useCreateTask.ts

```diff
@@ -0,0 +1,60 @@
+import { useQueryClient } from '@tanstack/react-query'
+import { useForm } from 'react-hook-form'
+import { useToast } from '@/_hooks'
+import { useAuthStore } from '@/_stores'
+import { isErrorWithMessage } from '@/_utils'
+import { useCreateTaskMutation } from './api'
+import { createTaskResolver, taskSchema } from '../schema'
+
+export const useCreateTask = () => {
+  const queryClient = useQueryClient()
+  const getHouseId = useAuthStore((state) => state.getHouseId)
+  const houseId = getHouseId()
+  const { toast } = useToast()
+
+  const form = useForm<taskSchema>({
+    resolver: createTaskResolver,
+    defaultValues: {
+      title: '',
+      date: undefined,
+      assigneeId: undefined,
+      note: '',
+    },
+  })
+
+  const { mutate, isPending } = useCreateTaskMutation()
+
+  const handleSuccess = () => {
+    // TODO: Need to invalidate queryKey that is related to tasks
```

Comment:
I think GetMe?

Reply:
I'm not sure yet but you might add some query keys in the future when making calendar.
If you just use GetMe, I can just add that query

Reply:
Oh sorry I thought you're using GetMe for top page but it was not. Never mind! You can just leave it like this